src/deepdisc/astrodet/astrodet.py

```diff
@@ -99,6 +99,7 @@
 from tabulate import tabulate
 from torch.nn.parallel import DistributedDataParallel
 
+from deepdisc.astrodet import detectron as detectron_addons
```

Comment:
Not a blocker - I'm not immediately seeing why this line was added.

Reply:
Wasn't sure if I should explicitly mention it (nice eye there)! 

It should have been added in the process of combining various versions of `train_mapper_cls` and moving it into astrodet. The previous versions had all lived in places that had this import, and I had overlooked that in the astrodet PR.